onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -445,6 +473,34 @@ def aten_any_dim(self: TTensor, dim: int, keepdim: bool = False) -> BOOL:
     return result
 
 
+@torch_op("aten::any.dims", trace_only=True)
+def aten_any_dims(self: TTensor, dim: Sequence[int] = (), keepdim: bool = False) -> BOOL:
+    """any.dims(Tensor self, int[1]? dim=None, bool keepdim=False) -> Tensor"""
+
+    if not dim:
+        return aten_any_dims_empty_dim(self, keepdim)
+    for d in dim:
+        self = aten_any_dim(self, d, keepdim)
+    return self
+
+
+@torch_op("aten::any.dims")
+def aten_any_dims_empty_dim(self: TTensor, keepdims: bool) -> BOOL:
```

Comment:
nit:

Change 'empty' to 'no' to avoid confusion that users might think dims is an empty list/tuple where it is actually None.

Reply:
Done

Reply:
dims can be an empty list or None. They will mean the same thing. I made the changes nevertheless.